doc/development/plugins.rst

```diff
@@ -85,11 +85,14 @@ Device capabilities
 You must further tell PennyLane about the operations that your device supports, 
 as well as potential further capabilities, by providing the following class attributes/properties:
 
-* :attr:`.Device.operations`: a set of the supported PennyLane operations as strings, e.g.,
-
-  .. code-block:: python
-
-    operations = {"CNOT", "PauliX"}
```

Comment:
Maybe worth keeping this example for the 'not inherited from `default.qubit`' case?

Reply:
added back in.